src/function.rs

```diff
@@ -250,10 +251,47 @@ impl<'lua> Function<'lua> {
 
         data
     }
+
+    /// Returns source and line number information about the function.
+    ///
+    /// Corresponds to the `>S` what mask for `lua_getinfo` when applied
+    /// to the function.
+    #[cfg(not(feature = "luau"))]
```

Comment:
I was unable to figure out how to get this to work with `luau`, it's missing the `lastlinedefined` element and I get panics in `mlua_assert` when I conditionally enable it for `luau`.